app/services/expert_reminder_service.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+class ExpertReminderService
+  attr_accessor :experts_hash
+
+  class << self
+    def send_reminders
+      @experts_hash = {}
+      selected_assistances_experts_needing_taking_care_update
```

Comment:
I would rather have methods that return the hash into send°reminders 
I don't like collateral modifications like those

Reply:
I'm OK if you're motivated to improve it :D